packages/telemetry/lib/pluggable-instrumentations.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+// These are already set automatically by the runtime, so we throw
+// if set again.
+const defaultInstrumentations = [
+  '@opentelemetry/instrumentation-pg',
+  '@opentelemetry/instrumentation-http',
+  '@opentelemetry/instrumentation-undici'
+]
+
+const getInstrumentationInstance = async (instrumentationConfig) => {
+  if (typeof instrumentationConfig === 'string') {
+    instrumentationConfig = { package: instrumentationConfig, exportName: 'default', options: {} }
+  }
+  const { package: packageName, exportName = 'default', options = {} } = instrumentationConfig
+
+  if (defaultInstrumentations.includes(packageName)) {
+    throw new Error(`Instrumentation package ${packageName} is already included by default, please remove it from your config.`)
+  }
+
+  let mod
+  try {
+    mod = await import(packageName)
```

Comment:
This will not work in practice. You have to do https://github.com/platformatic/platformatic/blob/afa9319e77ed6c2dcba7f9fb3fded896786b2e70/packages/create-platformatic/src/index.mjs#L55-L62 otherwise you might not be able to load them in certain edge cases or package managers.